### PR TITLE
Local feedpage antdesign

### PR DESCRIPTION
**Pull request by danielcfuentes**
## Description

what this pull request is doing:
This pull request migrates the the whole Feedpage and its components like the Add button, Modal, and etc.
In addition created a function to generate a users avatar color based on thier name.

what will be done in later PRs and not included here:
Start working on the routes page

## Milestones
finished Milestone 4: React UI Components

## Test Plan
https://www.loom.com/share/2a67d1315e0a4f218a6200ebef6e8196?sid=d4a80ba7-0f74-419f-9078-4d03533577fe

### CODE REVIEW

File: frontend-runitup/src/utils/apiConfig.js [hunk 1]
```diff
@@ -17,3 +17,12 @@ export const getAuthHeaders = () => {
     // Don't include 'Content-Type' here as it will be set automatically for FormData
   };
 };
+
+export const generateColor = (name) => {
```
Comment (janicetsai1) [inline frontend-runitup/src/utils/apiConfig.js:21]: this is so cool!

**Review by janicetsai1 — APPROVED** [review]

File: frontend-runitup/src/utils/apiConfig.js (hunk 1 above)
Comment (danielcfuentes) [reply frontend-runitup/src/utils/apiConfig.js:21]: Thank you!

Comment (gwh59) [reply frontend-runitup/src/utils/apiConfig.js:21]: yeah ditto this is rad

**Review by gwh59 — COMMENTED** [review]
lgtm!

**Pull request merged**